pygluu/kubernetes/templates/helm/gluu/values.yaml

```diff
@@ -135,6 +135,8 @@ global:
   oxauth:
     # -- Boolean flag to enable/disable oxauth chart. You should never set this to false.
     enabled: true
+    # -- passing custom  java options to oxauth.
+    gluuCustomJavaOptions: "-XX:MaxDirectMemorySize=2500m -Xmx2200m"
```

Comment:
The number seems based on default `resources.limits.`.

@moabu do we want user to override this setting manually or pre-calculated from its `resources.limits.` attributes ?

Reply:
Pre calculate based on each services resource limits. Otherwise , this will defeat the purpose of the resource limits defined .